### PR TITLE
As item view

### PR DESCRIPTION
**Pull request by SashaShostyr**

### CODE REVIEW

**Review by Vadser — APPROVED** [review]

File: app/views/items/show.html.slim [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
-  = @item.category&.name
-p
-  strong
-  - if @item.free
-    strong THIS ITEM IS FREE
-= link_to 'Edit', edit_item_path(@item)
-| \
-= link_to 'Back', items_path
+
+p#notice= notice
```
Comment (simonovich) [inline app/views/items/show.html.slim]: Maybe move this notice to layout?

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -44,6 +44,13 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.after(:all) do
+    if Rails.env.test?
```
Comment (simonovich) [inline spec/rails_helper.rb]: Try it 'return unless'

File: app/helpers/slider_helper.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+module SliderHelper
+  def indicators(images)
+    result = ''
+    0.upto(images.count - 1) do |counter|
```
Comment (anatoliliotych) [inline app/helpers/slider_helper.rb]: how about inject?

File: app/helpers/slider_helper.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      result << "<li class='#{class_active(counter)}' data-slide-to='#{counter}' data-target='#myCarousel'></li>"
+    end
+    result
+  end
+
+  def slides(images)
+    result = ''
+    images.each_with_index do |image, index|
```
Comment (anatoliliotych) [inline app/helpers/slider_helper.rb]: how about inject?

**Review by anatoliliotych — CHANGES_REQUESTED** [review]

**Review by anatoliliotych — APPROVED** [review]

**Pull request merged**